sdk/storage/inc/datalake/path_client.hpp

```diff
@@ -196,40 +114,22 @@ namespace Azure { namespace Storage { namespace DataLake {
      *             permissions.
      * @param options Optional parameters to set an access control to the resource the path points
      *                to.
-     * @return PathFlushDataResponse
+     * @return Azure::Core::Response<PathSetAccessControlInfo>
+     * @remark This request is sent to dfs endpoint.
      */
-    PathSetAccessControlResponse SetAccessControl(
+    Azure::Core::Response<PathSetAccessControlInfo> SetAccessControl(
         std::vector<Acl> acls,
         const SetAccessControlOptions& options = SetAccessControlOptions()) const;
 
-    /**
-     * @brief Sets POSIX access control rights on files and directories under given path
-     *        recursively.
-     * @param mode Mode PathSetAccessControlRecursiveMode::Set sets POSIX access control rights on
-     *             files and directories, PathSetAccessControlRecursiveMode::Modify modifies one or
-     *             more POSIX access control rights  that pre-exist on files and directories,
-     *             PathSetAccessControlRecursiveMode::Remove removes one or more POSIX access
-     *             control rights that were present earlier on files and directories
-     * @param acls Sets POSIX access control rights on files and directories. Each access control
-     *             entry (ACE) consists of a scope, a type, a user or group identifier, and
-     *             permissions.
-     * @param options Optional parameters to set an access control recursively to the resource the
-     *                path points to.
-     * @return PathSetAccessControlRecursiveResponse
-     */
-    PathSetAccessControlRecursiveResponse SetAccessControlRecursive(
-        PathSetAccessControlRecursiveMode mode,
-        std::vector<Acl> acls,
-        const SetAccessControlRecursiveOptions& options = SetAccessControlRecursiveOptions()) const;
-
     /**
      * @brief Sets the properties of a resource the path points to.
      * @param options Optional parameters to set the http headers to the resource the path points
      * to.
-     * @return SetPathHttpHeadersResponse
+     * @return Azure::Core::Response<SetPathHttpHeadersInfo>
+     * @remark This request is sent to blob endpoint.
      */
-    SetPathHttpHeadersResponse SetHttpHeaders(
-        Azure::Storage::DataLake::DataLakeHttpHeaders httpHeaders,
+    Azure::Core::Response<SetPathHttpHeadersInfo> SetHttpHeaders(
```

Comment:
This response name is kind of strange.

Reply:
Well, I can make it `SetHttpHeadersInfo` but I doubt that this is the place where you feel strange about.